deploy-board/deploy_board/webapp/host_views.py

```diff
@@ -19,6 +19,9 @@
 from helpers import environs_helper, agents_helper, autoscaling_groups_helper
 from helpers import environ_hosts_helper, hosts_helper
 from deploy_board.settings import IS_PINTEREST, TELETRAAN_HOST_INFORMATION_URL
+import brood
```

Comment:
for a simpler fix, I would suggest just add IS_PINTEREST around it, and hide the part uses brood function as well ( probably the part show host details, the opensource version does not have this feature any ways)